web/components/MiniRoadmap.tsx

```diff
@@ -39,58 +41,76 @@ export const MiniRoadmap = (props: Props): ReactElement => {
     }
   };
 
-  const isLast = (stepNumber: number) => {
-    return roadmap?.steps[roadmap?.steps.length - 1].step_number === stepNumber;
+  const getSection = (sectionName: SectionType) => {
+    const _sectionName = sectionName.toLowerCase();
```

Comment:
what is starting a variable name with an underscore intended to convey? Haven't seen that pattern